bashate fails in master branch

it fails with:

```text
.github/motives.sh:24:1: E020 Function declaration not in format ^function name {$
.github/motives.sh:34:1: E020 Function declaration not in format ^function name {$
.github/motives.sh:44:1: E020 Function declaration not in format ^function name {$
```

Let's fix it

--- a/.github/motives.sh
+++ b/.github/motives.sh
@@ -21,7 +21,7 @@ while IFS= read -r f; do
   echo "${n} -> $(du -b "${html}" | cut -f1) bytes"
 done < <(find src/main/resources/org/eolang/motives -name '*.md')
 
-function list_them() {
+list_them() {
   printf "<ul>\n"
   while IFS= read -r f; do
     n=$(basename "${f}" ".html")
@@ -31,7 +31,7 @@ function list_them() {
   printf '</ul>\n'
 }
 
-function head() {
+head() {
   printf '<html>\n'
   printf '<head>\n'
   printf '<meta charset="UTF-8"/>\n'
@@ -41,7 +41,7 @@ function head() {
   printf '<body><section><article>\n'
 }
 
-function tail() {
+tail() {
   printf '<p>Published on %s.</p>\n' "$(date)"
   printf '</article></section></body></html>'
 }

--- a/.github/workflows/bashate.yml
+++ b/.github/workflows/bashate.yml
@@ -1,24 +1,5 @@
-# The MIT License (MIT)
-#
-# Copyright (c) 2016-2025 Objectionary.com
-#
-# Permission is hereby granted, free of charge, to any person obtaining a copy
-# of this software and associated documentation files (the "Software"), to deal
-# in the Software without restriction, including without limitation the rights
-# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-# copies of the Software, and to permit persons to whom the Software is
-# furnished to do so, subject to the following conditions:
-#
-# The above copyright notice and this permission notice shall be included
-# in all copies or substantial portions of the Software.
-#
-# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
-# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-# SOFTWARE.
+# SPDX-FileCopyrightText: Copyright (c) 2016-2025 Objectionary.com
+# SPDX-License-Identifier: MIT
 ---
 name: bashate
 'on':

